### ISSUE TITLE
If txo spend (tip unlock) fails, show error

### ISSUE THREAD

**Issue by tzarebczan** [lbryio/lbry-desktop#5728]
Right now it just hands in pending state with no errors.

**Comment by infinite-persistence** [comment]
- Is that the one in the Wallet Page?  The error message does appear (tested by hardcoding bad claim ID | unplug cable).
- Also tried the one in the File Page, and the error message is handled.

Maybe there's a specific scenario that slipped.

**Comment by tzarebczan** [comment]
Yup, the main tip unlock. Mine failed on the sdk side due to missing private keys, and there was no error. Not sure show to simulate a failure, but i know that the tip unlock on the content pages does surface it.
